src/styles/header.scss

```diff
@@ -0,0 +1,84 @@
+:root {
+  --active-color: #00acdc;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  width: 100%;
+  padding: 0 50px;
+  box-sizing: border-box;
+  box-shadow: 0 2px 4px 0 rgba(0, 0, 0, 0.05);
+}
+
+.logo-pic {
+  margin: 10px 0 6px;
+  height: 40px;
+}
+
+.list {
+  list-style-type: none;
+  display: flex;
+  margin: 0;
+  padding: 0;
+  justify-content: center;
+}
+
+.content {
```

Comment:
this is a separate bem blocks, you should make them as elements or move them to the separate files